src/test/java/com/sap/cloudfoundry/client/facade/ApplicationsCloudControllerClientIntegrationTest.java

```diff
@@ -79,12 +80,15 @@ void createApplication() {
                                           .healthCheckHttpEndpoint(HEALTH_CHECK_ENDPOINT)
                                           .healthCheckTimeout(HEALTH_CHECK_TIMEMOUT)
                                           .build();
-        CloudRouteSummary cloudRouteSummary = ImmutableCloudRouteSummary.builder()
-                                                                        .host(APPLICATION_HOST)
-                                                                        .domain(DEFAULT_DOMAIN)
-                                                                        .build();
+        CloudRoute route = ImmutableCloudRoute.builder()
+                                              .host(APPLICATION_HOST)
+                                              .domain(ImmutableCloudDomain.builder()
+                                                                          .name(DEFAULT_DOMAIN)
+                                                                          .build())
+                                              .url(APPLICATION_HOST+"."+DEFAULT_DOMAIN)
```

Comment:
Not formatted

Reply:
This is an error with GitHub's visualization. Both in IntelliJ and in VS Code, this section looks formatted correctly.